### ISSUE TITLE
vibrating valve and standpipe

### ISSUE THREAD

**Issue by osrf-migration** [osrf/drcsim#298]
**Original report ([archived issue](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/issues/298)) by Maurice Fallon (Bitbucket: [mauricefallon](https://bitbucket.org/%7B8bca6a99-adab-444a-b7f0-81660c79adfb%7D/)).**

----------------------------------------
We see vibrating valves and standpipes in our current version:
[http://youtu.be/QXcuCWUvWEs](http://youtu.be/QXcuCWUvWEs)
The vibration is not always present - it seems to depend on initial conditions.

@gerkey @nkoenig Can the valve and standpipe be shorted to hang a few mm short of the wall?

**Comment by osrf-migration** [comment]
**Original comment by Thomas Koletschka (Bitbucket: [thomasko](https://bitbucket.org/%7Bb33092ac-6376-48d1-9803-d9e6de445d1a%7D/)).**

----------------------------------------
Huh, I've never seen this since the last release where valve and standpipe got moved away from the wall to not penetrate it. Did you accidentally move the wall or something like that? :P

**Comment by osrf-migration** [comment]
**Original comment by Steve Peters (Bitbucket: [Steven Peters](https://bitbucket.org/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D/), GitHub: [scpeters](https://github.com/scpeters)).**

----------------------------------------
I thought we had fixed this in [pull request #299](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/pull-requests/299). Can you verify that you're using the latest version?

**Comment by osrf-migration** [comment]
**Original comment by Thomas Koletschka (Bitbucket: [thomasko](https://bitbucket.org/%7Bb33092ac-6376-48d1-9803-d9e6de445d1a%7D/)).**

----------------------------------------
When I compare the pattern on your floor with the one in my world it seems like your wall is closer to the table than in my world, so you might either be running some older version or it got moved.

@kennysharma suggested for you to clear out your .gazebo/models dir as this happened to him as well the first time he upgraded to 2.6.1 and clearing seemed to fix it (maybe the wall thickness got decreased at some point?).

**Comment by osrf-migration** [comment]
**Original comment by John Hsu (Bitbucket: [hsu](https://bitbucket.org/%7B0a186eae-abf0-4514-a951-23db5eccc286%7D/), GitHub: [hsu](https://github.com/hsu)).**

----------------------------------------
Hi @faller,

Are you compiling from source or running from debian install?  I just tested drcsim-nightly and it does not have the this issue.

Can you clarify what you meant by "it seems to depend on initial conditions"?  Are you moving the objects around?

Thanks,
John

**Comment by osrf-migration** [comment]
**Original comment by Maurice Fallon (Bitbucket: [mauricefallon](https://bitbucket.org/%7B8bca6a99-adab-444a-b7f0-81660c79adfb%7D/)).**

----------------------------------------
@hsu: I just flushed my system again and installed the regular drcsim apt package. 
And then it pulled down the vrc task 3 models. And it worked fine.

What I did before was manually install the models from mercurial. I think the head of the gazebo_models is not compatiable with the VRC arenas

**Comment by osrf-migration** [comment]
**Original comment by Thomas Koletschka (Bitbucket: [thomasko](https://bitbucket.org/%7Bb33092ac-6376-48d1-9803-d9e6de445d1a%7D/)).**

----------------------------------------
I'm using the head of ``gazebo_models`` and it works like a charm ;)

**Comment by osrf-migration** [comment]
**Original comment by John Hsu (Bitbucket: [hsu](https://bitbucket.org/%7B0a186eae-abf0-4514-a951-23db5eccc286%7D/), GitHub: [hsu](https://github.com/hsu)).**

----------------------------------------
@faller:

If you can confirm default tip ([gazebo_models 305:95886143b61e](https://bitbucket.org/osrf/gazebo_models/commits/95886143b61e))?

~~~
$ hg sum
parent: 305:95886143b61e 
 Merged in pioneer3at ([pull request #46](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/pull-requests/46))
branch: default
commit: 9 deleted, 149 unknown (clean)
update: (current)
mq:     (empty queue)
~~~

works fine, we can close this issue.

Thanks,

John

**Comment by osrf-migration** [comment]
**Original comment by Brian Gerkey (Bitbucket: [Brian Gerkey](https://bitbucket.org/%7B3d8b5763-9dad-4be7-85c2-a7ab6cd49173%7D/), GitHub: [gerkey](https://github.com/gerkey)).**

----------------------------------------

* changed state from "new" to "resolved"

Resolved by update.  Filed [related ticket](https://bitbucket.org/osrf/gazebo_models/issue/11) to consider branch gazebo_models.
